uncharted-aske/HMI
# Gromet structural comparison

**What**

- Added CHIME+ in PNC Gromet format for testing purposes (this should be replaced once it is provided by donu)
- Adjusted the Comparison view so it shows the results of the comparison between SIMPLE SIR and CHIME+ in PNC format.
- This is also shown in the Simulation view so you can compare them structurally as well as behaviorally (parameters and variables can be inspected at the same time)
- Adjusted the ModelPanel from the Simulation view for comparison purposes so we don't show the expand button at all times.

**Testing**

- Select SimpleSIR_metadata and CHIME+ from the list of models.
- You should be able to visualize the comparison between models (marked in pink-ish)
- Open the simulation space, it should still be visible there.

NOTE: I still need to figure out the highlighting when you select a node (it should be highlight the corresponding node in the other graph but I still need to think about the best data structure).

https://user-images.githubusercontent.com/10552785/128778479-6198545c-1007-4a1b-97b9-cf1084f89ffb.mov
